packages/react/src/use-feature-flag.ts

```diff
@@ -1,51 +1,36 @@
-import { Client, EvaluationDetails, FlagValue, ProviderEvents, ResolutionDetails } from '@openfeature/web-sdk';
+import { Client, EvaluationDetails, FlagValue, ProviderEvents } from '@openfeature/web-sdk';
 import { useEffect, useState } from 'react';
 import { useOpenFeatureClient } from './provider';
 
 export function useFeatureFlag<T extends FlagValue>(flagKey: string, defaultValue: T): T {
-  const [flagEvaluationDetails, setFlagEvaluationDetails] = useState<ResolutionDetails<T> | null>(null);
+  const [, setForceUpdateState] = useState({});
```

Comment:
```suggestion
  const [setForceUpdateState] = useState({});
```

Reply:
Mh this does not work isn't it?
It would make the state be setState if I am not totally overseeing something @toddbaert🤔

Reply:
Oh, I'm sorry, I misread. I though this was a typo, but I see it's an array de-structure now.